src/Kernel/Context.class.st

```diff
@@ -1778,6 +1778,17 @@ Context >> runUntilErrorOrReturnFrom: aSender [
 	].
 ]
 
+{ #category : #'debugger access' }
+Context >> searchDoIt [
```

Comment:
There are other search methods in context but with the word #lookup. I would follow this name here: #lookupDoItContext